src/Middleware/PaginationHeaderTag.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Yiisoft\Yii\Rest\Middleware;
+
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+
+/**
+ * TODO should be moved to yii-web
+ */
+final class PaginationHeaderTag implements MiddlewareInterface
```

Comment:
What is the use case for it in the web package?

Reply:
Shouldn't `X-Pagination-*` headers be in the web page?

Reply:
What for?

Reply:
For non-REST API.
@roxblnfk suggested move all middlewares to the separate package.

Reply:
I don't think it should be moved. There is no use for it in web app.